core/server/common/src/main/java/alluxio/master/AbstractMaster.java

```diff
@@ -61,7 +61,7 @@
   /** A handler to the journal for this master. */
   private Journal mJournal;
   /** true if this master is in leader mode, and not standby mode. */
-  private boolean mIsLeader = false;
+  protected boolean mIsLeader = false;
```

Comment:
NIT: instead of using `protected` member variable and let subclasses reference it directly, I suggest to have provide a getter `isLeader()` and let subclasses call the getter